frontend_app_admin_workspace_user/src/css/index.styl

```
@@ -146,14 +146,19 @@
     & .table
       &__id
         width 5%
-        text-align right
+        text-align center
+      &__type
+        width 10%
+        text-align center
+        & i
+          margin-right 5px
```

```suggestion
          margin-inline-end 5px
```
This will help support right to left languages in the future

(same for margin-left → margin-inline-start)